### PR TITLE
Added link for "adding descriptive tags"

### LINKED ISSUE #2228: Link for 'adding descriptive tags' not working .
Hi, this is a [first-timers-only](https://github.com/publiclab/plots2/projects/2) issue. This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

If you have contributed before, consider leaving this one for someone new, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

# The Problem

In the following image , the link **adding descriptive tags below.** is not working , 

screen shot 2018-02-04 at 11 52 26 pm

Lets **move to the bottom of the page** when we click this link .

# Solution

The line of code to change is : 
https://github.com/publiclab/plots2/blob/8f60c60d68aa60867f4edbef49f03f91e63875cd/config/locales/views/wiki/show/en.yml#L6

Lets add this to move to bottom of the page : 
`<a href='javascript: document.body.scrollIntoView(false);'>adding descriptive tags below</a>.`

### Feel free to ask any doubt regarding the issue ! Thank you !

# Steps to Fix 

- [ ] claim this issue with a comment here, below, and ask any clarifying questions you need
- [ ]  set up a repository locally following the README instructions, and make sure that all tests pass
 - [ ] try to fix the issue following the steps above, but even before you're done, you can:
 commit your changes and start a pull request (see contributing to Public Lab software) but mark it as "in progress" if you have questions or if you haven't finished
- [ ]  alert someone via the developers list (see below) to have your pull request merged. This may take a few extra steps depending on when you do it, but we'll help you out!

Please email the developers list (see [https://publiclab.org/wiki/developers](https://publiclab.org/wiki/developers)) or go to the [chatroom](http://publiclab.org/chat) if you have questions, and take a look at our [first-timers landing page for more information!](https://github.com/publiclab/plots2/projects/2)

### PR DESCRIPTION
**Pull request by g33kyshivam**
Fixes #2228
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] all tests pass -- `rake test:all`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request is descriptively named with #number reference back to original issue

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

**Comment by sagarpreet-chadha** [comment]
Hi @g33kyshivam , have you tried the solution given [here](https://stackoverflow.com/questions/15859113/focus-not-working) .

File: config/locales/views/wiki/show/en.yml [hunk 1]
```diff
@@ -3,8 +3,8 @@
 en:
   wiki:
     show:
-     page_has_no_tags: "This page has no tags yet; help connect it to other content by <a onClick='$('#taginput').focus()'>adding descriptive tags below</a>."
-     revision_from: "This is a revision from %{time}."
+     page_has_no_tags: "This page has no tags yet; help connect it to other content by <a href='javascript: document.querySelector('.tag-input').focus();'>adding descriptive tags below</a>"
```
Comment (seafr) [inline config/locales/views/wiki/show/en.yml]: The string seems to have single quotes nested inside single quotes.

File: config/locales/views/wiki/show/en.yml [hunk 2]
```diff
@@ -3,8 +3,8 @@
...
+     revision_from: "This is a revision from %{time}."
```
Comment (seafr) [inline config/locales/views/wiki/show/en.yml]: This line has trailing spaces.

File: config/locales/views/wiki/show/en.yml (hunk 1 above)
Comment (jywarren) [reply config/locales/views/wiki/show/en.yml]: Yes, good catch! So, can the inner ones be escaped?

Comment (seafr) [reply config/locales/views/wiki/show/en.yml]: Maybe we can use single quotes on the outer string, then double quotes on the JS string, and finally *escaped* single quotes on the innermost string like this: 
`'... <a href="javascript: document.querySelector(\'.tag-input\').focus();">adding ...'`
What do you think @jywarren?

Comment (jywarren) [reply config/locales/views/wiki/show/en.yml]: That sounds good. @g33kyshivam want to give this a try?

**Comment by jywarren** [comment]
Hi, just checking if you've gotten stuck or were able to wrap this up -- it's very close! We'd love your help. 👍

**Comment by Gauravano** [comment]
hi @jywarren , this one is ready to be merged. Thanks. 

shivam_pr

**Comment by jywarren** [comment]
Awesome!!!! Thanks all!!

**Pull request merged**